### PR TITLE
[PIE-3153][Martin] Checkbox to show or hide weekends from attendance view

### PR DESCRIPTION
**Pull request by MartinM1312**
[PIE-3153][Martin] Checkbox to show or hide weekends from attendance view

## 💅🏼 What issue does this fix?
Checkbox to show or hide weekends from attendance view

## 🍂 Before You Submit

* [X] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Migrations
* [ ] Dependencies

### CODE REVIEW

File: client/src/AttendanceView/AttendanceView.js [hunk 1]
```diff
@@ -136,11 +145,20 @@ export function AttendanceView() {
...
+
+  useEffect(() => {
+    setColumns(generateColumns())
+    // eslint-disable-next-line
+  }, [showWeekends])
+
   // create seven columns for each day of the week
   const generateColumns = () => {
     const dateColumns = []
+    var i = showWeekends ? 0 : 1
```
Comment (fabianraf) [inline client/src/AttendanceView/AttendanceView.js:158]: Don't use var. You should be able to use let instead.

File: client/src/AttendanceView/AttendanceView.js [hunk 2]
```diff
@@ -574,7 +592,7 @@ export function AttendanceView() {
               {t('inputAttendance')}
             </Button>
           </div>
-          <div className="flex">
+          <div className="flex" style={filtersContainerStyle}>
```
Comment (fabianraf) [inline client/src/AttendanceView/AttendanceView.js:595]: Don't use inline styles. You should add classes to the css.

File: client/src/AttendanceView/AttendanceView.js (hunk 1 above)
Comment (MartinM1312) [reply client/src/AttendanceView/AttendanceView.js]: Done

File: client/src/AttendanceView/AttendanceView.js (hunk 2 above)
Comment (MartinM1312) [reply client/src/AttendanceView/AttendanceView.js]: Done

**Review by fabianraf — APPROVED** [review]

**Pull request merged**